domains/lordstickman543.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "I want a base of operations for my programming stuff. my website kinda sucks but I am trying to figure out CSS so later it will be better",
+    "repo": "https://github.com/lordstickman543/website",
+    "owner": {
+        "username": "lordstickman543",
+        "email": "lordstickman543@gmail.com"
+    },
+    "record": {
+        "CNAME": "lordstickman543.github.io/website"
```

```suggestion
        "CNAME": "lordstickman543.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).